pact-wrapper/modules/au.com.dius.pact.consumer.dsl/DslPart.bal

```diff
@@ -0,0 +1,39 @@
+// This is an empty Ballerina class autogenerated to represent the `au.com.dius.pact.consumer.dsl.DslPart` Java class.
```

Comment:
All the files in the `modules` folder seem to be partially generated ones from the bindgen. Can we remove these as well?